src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    white,
+    black,
+    orange,
+    green,
+    blue,
+    purple,
+    yellow

```

Comment:
Enum constants should be in uppercase. The Java naming convention for constants uses all uppercase letters with words separated by underscores.